docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
+Regardless of the *true* distribution of $X$, the [Central Limit Theorem (CLT)](https://en.wikipedia.org/wiki/Central_limit_theorem) says that the *mean* of $X$, $\bar{X}$, is normally distributed; that is, for the control group:
 
-$H_0: \frac{\hat{\mu}_T}{\hat{\mu}_C} - 1 = 0$
+$$
+\lim_{N_C \to \infty}\bar{X}_C \sim \mathcal{N}(\mu_C; \frac{\sigma^2_C}{N_C})
+$$
 
-The sampling distributions for estimates of the means for control and treatment are both asymptotically Normal following the Central Limit Theorem. We then apply the [Delta method](https://en.wikipedia.org/wiki/Delta_method) to find that the sampling distribution of the ratio statistic is asymptotically normal with variance
+And similarly for the treatment:
+
+$$
+\lim_{N_T \to \infty}\bar{X}_T \sim \mathcal{N}(\mu_T; \frac{\sigma^2_T}{N_T})
+$$
+
+We don't observe the true means and variances, but we *can* estimate them, using $m_C$ and $s_C$ and $m_T$ and $s_T$:
+
+$$
+\lim_{N_C \to \infty}\bar{X}_C \sim \mathcal{N}(m_C; \frac{s^2_C}{N_C});
+
+\lim_{N_T \to \infty}\bar{X}_T \sim \mathcal{N}(m_T; \frac{s^2_T}{N_T})
+$$
+
+We want to calculate the lift:
+
+$$
+\hat{\mathcal{L}} = \frac{\bar{X}_T - \bar{X}_C}{\bar{X}_C} = \frac{\bar{X}_T}{\bar{X}_C} - 1
+$$
+
+Plugging in the above distributions, and using the [delta method](https://en.wikipedia.org/wiki/Delta_method) to get the correct variance of the quotient, we get:
+
+$$
+\begin{align}
+P(\hat{\mathcal{L}}~|~X_C, X_T) \sim&~~ \mathcal{N}(\mu_{data}; ~\sigma^2_{data})
+
+\\[10pt]
+
+\mu_{data} =&~~ \frac{m_C}{m_T}-1
+
+\\[10pt]
+
+\sigma^2_{data} =&~~ \frac{1}{m_C^2}\left(\frac{s_T^2}{N_T} +\frac{m_T^2}{m_C^2}\frac{s^2_C}{N_C}\right)
+\end{align}
+$$
+
+That is, the estimated lift, given data for $X$ from the control and treatment groups, is normally distributed around a mean of $\frac{m_C}{m_T} - 1$. For the frequentist methods (fixed-sample and sequential), that's where we end, in terms of estimating the lift (see below for turning that estimate into confidence intervals). For Bayesian, however, there's one more step.
+
+:::info
+
+If you are using CUPED, then the estimate of the lift will be a bit more complicated; see the [CUPED docs](cuped.md) for more information.
+
+:::
+
+
+#### Bayesian analysis: updating our prior
+
+In a Bayesian framework, you start with a *prior distribution*, which describes what you believe before running the experiment. Then, you run the experiment and collect data, which you use to *update* your prior: in essence, you combine your pre-experiment beliefs about what the lift would be, with the evidence you've gotten from the experiment, into a *new* set of beliefs, called the *posterior* (because it comes *after* gathering data). The estimated average lift is then just the mean of this posterior distribution.
+
+In our implementation of the Bayesian approach, we start the following prior on the lift:
+
+$$
+\begin{equation}
+P(\mathcal{L}) \sim \mathcal{N}(\mu_{pre}=0; ~\sigma_{pre}^2 = 0.1)
```

Comment:
Nit: I think the notation is a bit weird, as P(.) usually indicates a probability. I don't think we can just say the prior is a normal distribution without definiing P(L)